urth/dashboard/nbexts/bundle_handler.py

```diff
@@ -199,7 +199,7 @@ def _get_local_app(self, abs_nb_path):
 
         :param abs_nb_path:
         '''
-        md = self._create_app_bundle(abs_nb_path, bundle_root=self.nb_dir, overwrite=True)
+        md = self._create_app_bundle(abs_nb_path, bundle_root=self.nb_dir, overwrite=True, template_fn='local.tpl')
```

Comment:
What if local.tpl does not exist, which it doesn't in this PR or in the code base?